importer/importer_utils.py

```diff
@@ -12,20 +12,21 @@
 site_cache = {}
 
 
-def get_absolute_path(dir_name):
+def get_file_from_subdir(dir_name, file_name):
     """
-    Get the absolute path of a subdirectory inside current one.
+    Get the absolute path of a file in a subdir inside current dir.
 
     This works in relation to where this file is actually
     located, not the current working directory, to make sure
     the tests don't try to resolve the path in relation
     to their own working directory.
+
+    :param dir_name: name of subdirectory
```

Comment:
"name or relative path (from directory containing importer_utils.py) of subdirectory". At least there is nothing preventing you from passing that.